src/server/services/wms/qgswmsrenderer.h

```diff
@@ -218,7 +220,7 @@ namespace QgsWms
       /**
        * Checks WIDTH/HEIGHT values against MaxWidth and MaxHeight
         \returns true if width/height values are okay*/
```

Comment:
Need to fix this if it now returns void

Reply:
Indeed, I forgot to update the documentation for now (also for other methods too).